equinor/EchoComponents
# chore: #62087 - security update 10 02 2022 without storybook

## Prerequisites

Please make sure you can check the following boxes:


## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


### Description

Updating dependencies, except for:
- react-sortablejs - drops import error on build. Needs further investigation.
![image](https://user-images.githubusercontent.com/13872908/153369591-72c48b1c-9788-42c2-9445-e63a0cbae4b9.png)
Found only this related post in the matter:
https://stackoverflow.com/questions/70098371/react-rollup-name-is-not-exported-by-node-modules

- css / style / svg loaders:
Makes storybook break. Almost got it work in the following draft PR - need to adjust SVG loader. More work needed.
https://github.com/equinor/EchoComponents/pull/70

Before updates:
![image](https://user-images.githubusercontent.com/13872908/153371514-5c1bd967-cca8-4e78-8f4f-503d9bf7b414.png)

After updates:
![image](https://user-images.githubusercontent.com/13872908/153369984-167cdc1c-6ed8-4506-9847-61e532571aa8.png)

![image](https://user-images.githubusercontent.com/13872908/153371677-3d376372-ff0e-40fe-83fb-5c9d8bd35061.png)

### How to test:
- run storybook, check if everything works / looks properly
- run build - check if the build is still green
